lte/gateway/c/core/oai/lib/secu/secu_defs.h

```diff
@@ -84,4 +97,10 @@ int nas_stream_encrypt_eea2(
 int nas_stream_encrypt_eia2(
     nas_stream_cipher_t* const stream_cipher, uint8_t const out[4]);
 
+int derive_5gkey_gnb(const uint8_t* kamf, uint32_t ul_count, uint8_t* kgnb);
+
+int derive_5gkey_xres_star(
+    uint8_t* ck_ik, uint8_t* snn, uint8_t* rand, uint8_t* res,
+    uint8_t* xres_star);
+
```

Comment:
this is moved to subscriberdb, can be removed from here.

Reply:
comment addressed.
removed derive_5gkey_xres_star